ksqldb-engine/src/main/java/io/confluent/ksql/execution/pull/HARouting.java

```diff
@@ -382,19 +409,19 @@ private enum RoutingResult {
     STANDBY_FALLBACK
   }
 
-  private static class PartitionFetchResult {
+  private static class NodeFetchResult {
```

Comment:
This is needed so that we can return the exception as well in the routing result

Reply:
Make sense.